modules/mbedtls/Kconfig.deprecated

```
@@ -0,0 +1,58 @@
+# Copyright (c) 2025 BayLibre SAS
+# SPDX-License-Identifier: Apache-2.0
+
+config MBEDTLS_MD
+	bool "[DEPRECATED] Generic message digest layer."
```

```suggestion
	bool "[DEPRECATED] Generic message digest layer"
```
? 🙂 if you need to make other changes